### ISSUE TITLE
Feature Request: Tobacco

### ISSUE THREAD

**Issue by Hinaichigo** [d3athrow/vgstation13#34886]
It'd be cool if botany had tobacco you could harvest and then make into cigarettes or put into pipes and maybe chew it like gum.

[featurerequest]

**Comment by wotisjanitor** [comment]
CO-SIGNED ✍️✍️

**Comment by Eneocho** [comment]
👍🏻 Would be cool if tobacco did anything beyond just being flavour.

**Comment by Hinaichigo** [comment]
Since nicotine already exists maybe it could be added to cigarettes and given effects like reducing bleeding or something since it's a vasoconstrictior.

**Comment by ErrorsWindows** [comment]
thumbs up emoji smokin that pixel game peace pipe ong

![2022-10-27_19 58 28](https://github.com/vgstation-coders/vgstation13/assets/122955989/458c450a-8f34-4d11-8566-229822e6a91e)

**Comment by Hinaichigo** [comment]
On one hand I was thinking it'd be cool if nicotine slightly slowed bleeding (and it was also added to existing cigarettes). But on the other hand maybe wouldn't want to make it so people are always smoking for that mechanical reason. Or would we? I'm kinda ambivalent. Does seem cool in ghetto surgery or if someone's wounded smoking to slow the blood flow. Maybe just a little bit so that once in a while it does make a difference between going into crit or not, but not so much that it's something everyone does it all the time as a replacement for other chems.
Also having a "good reason" to smoke, in opposition to the "bad reason" (igniting plasma in the air), seems nice.

**Comment by Eneocho** [comment]
Yeah, it's a hard thing to conceptually balance, but I'd say a slight decrease in bleeding from nicotine would, while powergameable, not ruin balance, as the common ways to die are more brutal than just bleeding (shotguns, rockets, lasers, toolboxing, eswords, etc). And it would be an extra chem to add to the antibleeder combo.
